core/src/main/java/co/aikar/commands/Annotations.java

```diff
@@ -50,7 +52,7 @@
     }
 
     String getAnnotationValue(AnnotatedElement object, Class<? extends Annotation> annoClass, int options) {
-        Annotation annotation = getAnnotationRecursive(object, annoClass);
+        Annotation annotation = getAnnotationRecursive(object, annoClass, new ArrayList<>()); // TODO: Better collection impl?
```

Comment:
It can be `Collections.emptyList()`

Reply:
Collections.emptyList is immutable. That doesn't make sense for me.